daprdocs/content/en/reference/components-reference/supported-cryptography/local-storage.md

```diff
@@ -0,0 +1,38 @@
+---
+type: docs
+title: "Local storage"
```

Comment:
Thoughts on adding "(Built-in)" to all the built-in components to make it more clear?

```suggestion
title: "Local storage (Built-in)"
```

Reply:
We have not done this with all the other components. I thinks we only do this once pluggable components gets closer to stable.

Reply:
In this case, "Built-in" means that it uses the built-in crypto engine in Dapr. So Dapr sees the raw key material. Not sure if there's a better way to explain this, aside from docs

Reply:
@msfussell @ItalyPaleAle do you think using `builtin.yaml` instead of `generic.yaml` is what's breaking the PR? it worked for me locally 🤷‍♀️